pom.xml

```diff
@@ -192,8 +233,8 @@
                     <!-- JDBC connection parameters -->
                     <jdbc>
                         <driver>org.postgresql.Driver</driver>
-                        <url>jdbc:postgresql://localhost/dbname</url>
-                        <user>username</user>
+                        <url>jdbc:postgresql://localhost/postgres</url>
```

Comment:
hier bitte dbname stehen lassen. ich glaube, zu dieser anpassung steht etwas in der anleitung